components/footer.component.js

```diff
@@ -0,0 +1,25 @@
+export class FooterComponent {
+  #page;
+  #twitterIcon;
+  #facebookIcon;
+  #linkedInIcon;
+
+  constructor(page) {
+    this.#page = page;
+    this.#twitterIcon = this.#page.locator('//li[@class="social_twitter"]');
+    this.#facebookIcon = this.#page.locator('//li[@class="social_facebook"]');
+    this.#linkedInIcon = this.#page.locator('//li[@class="social_linkedin"]');
+  }
+
+  async clickTwitterIcon() {
```

Comment:
Well, we can declare function like this to re-use it but here just one line and the same logic for others

If u will discontinue use it - that's fine (for this case)

'cause calling


```suggestion
  await clickTwitterIcon();
```
instead of
```suggestion
  await footerComponent.twitterIcon.click();
```
has no so much benefits here

Reply:
fixed